circle.yml

```diff
@@ -19,9 +19,16 @@ dependencies:
       source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh &&
       export PATH="$HOME/miniconda/bin:$PATH" &&
       source activate test &&
-      conda install pytest &&
+      conda install pytest cookiecutter -c conda-forge &&
       conda install pytest-cookies -c conda-forge &&
-      conda install ciocheck -c conda-forge;
+      conda install versioneer -c conda-forge &&
+      conda install ciocheck -c conda-forge &&
+      conda update pip -c conda-forge;
```

Comment:
I'm just testing, I think that the version from default channel was causing  the error in previous builds, updating it to conda-forge channel seems to fix the error